src/main/java/it/gov/pagopa/payhub/auth/mypay/service/MyPayUsersService.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.payhub.auth.mypay.service;
+
+import it.gov.pagopa.payhub.auth.mypay.model.MyPayUser;
+import it.gov.pagopa.payhub.auth.mypay.repository.MyPayUsersRepository;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MyPayUsersService {
+
+  private final MyPayUsersRepository myPayUsersRepository;
+
+  public MyPayUsersService(MyPayUsersRepository myPayUsersRepository) {
+    this.myPayUsersRepository = myPayUsersRepository;
+  }
+
+  public void registerMyPayUser(String externalUserId, String fiscalCode, String firstName, String lastName, String email) {
```

Comment:
to avoid ambiguitoy, let's call it mappedExternalUserId
```suggestion
  public void registerMyPayUser(String mappedExternalUserId, String fiscalCode, String firstName, String lastName, String email) {
```